tasks/system_probe.py

```diff
@@ -270,6 +270,7 @@ def kitchen_prepare(ctx):
             ctx,
             packages=pkg,
             skip_object_files=(i != 0),
+            skip_linters=True,
```

Comment:
This was causing our C linter to run several times when executing `inv -e system-probe.kitchen-prepare`

Reply:
👍